python/pyspark/ml/feature.py

```diff
@@ -543,6 +543,71 @@ class IDFModel(JavaModel):
     .. versionadded:: 1.4.0
     """
 
+@inherit_doc
+class MaxAbsScaler(JavaEstimator, HasInputCol, HasOutputCol):
+    """
+    .. note:: Experimental
+
+    Rescale each feature individually to a common range [min, max] linearly using column summary
```

Comment:
It appears you've just copied the docstring and test from `MinMaxScaler`. It is not correct. Please could you follow the doc for the Scala version of `MaxAbsScaler`, and create an appropriate docstring test.